src/script.js

```diff
@@ -368,3 +369,9 @@ export function getCountry(): $Values<typeof COUNTRY> {
 export function getLang(): $Values<typeof LANG> {
   return getLocale().lang;
 }
+
+export function getShopperInsights(): boolean {
```

Comment:
we should name this `getShopperInsightsUsed` or `getRecommendedPaymentMethodsUsed`